lib/libutils/ext/include/compiler.h

```diff
@@ -53,6 +53,7 @@
 #define __rodata	__section(".rodata")
 #define __rodata_unpaged __section(".rodata.__unpaged")
 #define __early_bss	__section(".early_bss")
+#define __notrace	__attribute__((no_instrument_function))
```

Comment:
Sounds good, so I'll change to use that instead.
FYI, the linux kernel uses `notrace` with no leading underscores (which is not OK here because `compiler.h` may be used in user applications so underscores are needed to avoid potential conflicts).